src/middleware/packages/webfinger/service.js

```diff
@@ -1,15 +1,16 @@
-const urlJoin = require('url-join');
+const { MIME_TYPES } = require('@semapps/mime-types');
 
 const WebfingerService = {
   name: 'webfinger',
   settings: {
-    usersContainer: null,
-    domainName: null // If not set, will be extracted from usersContainer
+    baseUrl: null,
```

Comment:
The activitypub service(s) actually call it `baseUri` (`I` not `L`), should this be adjusted to match?

Reply:
Sometimes it's a `I`, sometimes a `L`. We need to uniformize this. I guess for the semantic web, it's rather `I`.